src/Python-Wave-Serial-Encoder.py

```diff
@@ -348,14 +359,14 @@ def percent(n, t):
 labelsx.place(x=percent(2, w_fsx), y=percent(2, h_fsx))
 combo.place(x=percent(2, w_fsx), y=percent(8, h_fsx))
 connection.place(x=percent(2, w_fsx), y=percent(16, h_fsx))
-connB.place(x=percent(64, w_fsx), y=percent(8, h_fsx))
-refreshB.place(x=percent(90, w_fsx), y=percent(8, h_fsx))
+connB.place(x=percent(10, w_fsx), y=percent(32, h_fsx))
+refreshB.place(x=percent(30, w_fsx), y=percent(32, h_fsx))
```

Comment:
Ciao @sarachiodo ,
we need to move a bit more right the refreshB widget, because on my Linux PC it overlaps the connB widget. I suggest x=percent(50, w_fsx) that seems working fine on my side on both Linux and Windows PCs.
Best Regards,
Carlo